### DISCUSSION TITLE
Question regarding GaussDefCmpw

### DISCUSSION THREAD

**Discussion by BrushXue** [fppimenta/rheoTool#41] [Q&A]
Hello. I have a question regarding GaussDefCmpw

I'm trying to implement CUBISTA within OpenFOAM's original `LimiterFunc`. The limiter is defined by
```
max(min(min(1.5*r, (3.0 + r)/4.0), 1.5), 0)
``` 
according to 
> A convergent and universally bounded interpolation scheme for the treatment of advection, MA Alves, 2003

But the benchmark result is very terrible. I'm just wondering that two methods are theoretically equivalent. Why the original `LimiterFunc` won't work and I have to use `GaussDefCmpw` you provided? Thanks.

**Comment by fppimenta** [comment]
GaussDefCmpw = Gauss + Deffered correction + Componentwise

Detailed explanation here: [section 2.5](https://www.sciencedirect.com/science/article/pii/S0377025716303329#sec0002).
